scala-setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
Going through `/usr/bin/env` is said to be more portable (for Nix in particular, I think)
```suggestion
#!/usr/bin/env sh
```

Reply:
```suggestion
#!/usr/bin/env bash
```
bash might be a better choice too (I tend to always use bash, and never had any issue or anyone complain - [this SO answer](https://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux/17072017#17072017) suggests to default to bash too).